### PR TITLE
Don't mutate fields you're not supposed to

### PR DESCRIPTION
**Pull request by FredTheDino**

### CODE REVIEW

**Comment by FredTheDino** [comment]
If people want to merge or change the error message feel free to do so - no need to ask me.

**Review by EMattfolk — COMMENTED** [review]
I kind of feel like the name and id should not be part of the instance in the first place. At the very least not bundled with the other data in the blob.

File: sylt-common/src/error.rs [hunk 1]
```diff
@@ -251,6 +252,9 @@ impl fmt::Display for RuntimeError {
             RuntimeError::UnknownField(obj, field) => {
                 write!(f, "Cannot find field '{}' on blob {:?}", field, obj)
             }
+            RuntimeError::ImmutableField(field) => {
+                write!(f, "Cannot mutate field '{}' - since it is immutable", field)
```
Comment (sornas) [inline sylt-common/src/error.rs:256]:
```suggestion
                write!(f, "Cannot mutate field '{}' since it is immutable", field)
```

**Comment by FredTheDino** [comment]
> I kind of feel like the name and id should not be part of the instance in the first place. At the very least not bundled with the other data in the blob.

Fight me!

**Pull request merged**